### PR TITLE
Feature/new lvl1 intro

### PR DESCRIPTION
**Pull request by tonifj**

### CODE REVIEW

File: Scripts/src/Script/Lvl1IntroCinematic.cpp [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+	const std::string FINISH_TRIGGER = "Finish";
+	const std::string SHOW_LIGHT_BALL = "ShowLightBall";
+	const std::string HIDE_LIGHT_BALL = "HideLightBall";
+	const std::string CAMERA_SHAKE = "CameraShake";
+	const std::string DIALOGUE_START = "DialogueStart";
+	const std::string DIALOGUE_FINISH = "DialogueFinish";
+	const std::string DISSOLVER_START = "DissolverStart";
+
+	//dialogue
+	const std::string LIGHTBALL_FIRST_DIALOGUE = "The heart of the forest, we have          arrived! we can save Niberry!";
```
Comment (yalania) [inline Scripts/src/Script/Lvl1IntroCinematic.cpp:42]: You may can remove this dialogue constants or maybe add some dialogue to the cinematic

File: Scripts/src/Script/LightBall.cpp [hunk 2]
```diff
@@ -806,10 +806,10 @@ void LightBall::SetLightballColor() const
...
-	interior_texture_LB->color_emissive = math::Lerp(interior_texture_LB->color_emissive, interior_final_agressive_color, min(color_percentatge, 1.f));
+	//interior_texture_LB->color_emissive = math::Lerp(interior_texture_LB->color_emissive, interior_final_agressive_color, min(color_percentatge, 1.f));
 	exterior_texture_LB->color = math::Lerp(exterior_texture_LB->color, exterior_final_agressive_color, min(color_percentatge, 1.f));
-	exterior_texture_LB->color_emissive = math::Lerp(exterior_texture_LB->color_emissive, exterior_final_agressive_color_emissive, min(color_percentatge, 1.f));
+	//exterior_texture_LB->color_emissive = math::Lerp(exterior_texture_LB->color_emissive, exterior_final_agressive_color_emissive, min(color_percentatge, 1.f));
 	small_texture_LB->color = math::Lerp(small_texture_LB->color, small_final_agressive_color, min(color_percentatge, 1.f));
-	small_texture_LB->color_emissive = math::Lerp(small_texture_LB->color, small_final_agressive_color, min(color_percentatge, 1.f));
+	//small_texture_LB->color_emissive = math::Lerp(small_texture_LB->color, small_final_agressive_color, min(color_percentatge, 1.f));
```
Comment (yalania) [inline Scripts/src/Script/LightBall.cpp:813]: Why these are commented?

File: Scripts/src/Script/CinematicCamera.cpp [hunk 3]
```diff
@@ -292,7 +292,7 @@ void CinematicCamera::Awake()
 void CinematicCamera::Start()
 {
 	SetMainCamera(false);
-	CreateCinematics();
+	//CreateCinematics();
```
Comment (yalania) [inline Scripts/src/Script/CinematicCamera.cpp:295]: you can uncomment this line.

File: Scripts/src/Script/AllyInteractionsScript.cpp [hunk 4]
```diff
@@ -311,7 +311,7 @@ void AllyInteractionsScript::PlayAnimation(DialogueFeeling feeling)
 		}
 		if (current_state == State::INTRO_REPEAT )
 		{
-			cinematic_camera->PlayCinematic(1);
+			//cinematic_camera->PlayCinematic(1);
```
Comment (yalania) [inline Scripts/src/Script/AllyInteractionsScript.cpp:314]: you can uncomment this line.

File: Scripts/src/Script/LightBall.cpp (hunk 2 above)
Comment (tonifj) [reply Scripts/src/Script/LightBall.cpp:813]: It gave some errors at the beginning
imagen

File: Scripts/src/Script/CinematicCamera.cpp (hunk 3 above)
Comment (tonifj) [reply Scripts/src/Script/CinematicCamera.cpp:295]: petition accepted

File: Scripts/src/Script/AllyInteractionsScript.cpp (hunk 4 above)
Comment (tonifj) [reply Scripts/src/Script/AllyInteractionsScript.cpp:314]: alrighty

File: Scripts/src/Script/Lvl1IntroCinematic.cpp (hunk 1 above)
Comment (tonifj) [reply Scripts/src/Script/Lvl1IntroCinematic.cpp:42]: We got a tutorial right after the intro, i think ill remove this and will be junt fine

**Review by RickyVimon — APPROVED** [review]
LGTM

**Review by yalania — APPROVED** [review]

**Pull request merged**